### ISSUE TITLE
Dark Theme

### ISSUE THREAD

**Issue by wi0001** [royalfig/the-capitalizer#31]
Button to enable a dark mode, where all the background on the rest of the page goes dark! 
Would also like to thank you for making this project as it's probably one of the best Capitalizer's and I use it quite often, so thank you for your effort to create a sleek and functional converter. Hopefully you don't mind my suggestions! 😄

**Comment by royalfig** [comment]
As in other comments, I'm working on a redesign. I'll probably add dark mode support. Though, for the sake of simplicity, I'll probably just have it respect your system's default. What do you think?

**Comment by ItIsSeven** [comment]
Yep this sounds perfect and sounds even better than the way I mentioned.
